src/main/java/org/example/Extractor.java

```diff
@@ -3,23 +3,50 @@
 import org.jsoup.Jsoup;
 import org.jsoup.nodes.Document;
 import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
 
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 
 public class Extractor {
     public List<Chair> getChairs(String url){
-
-        String role, name, email;
         List<Chair> chairs = new ArrayList<>();
-        //todo
+        try {
+            Document doc = Jsoup.connect(url).get();
+            Elements people = doc.select("tr");
+            String role, name, email;
+            for (Element individual : people) {
+                String title = individual.select("td").text();
+                boolean chair = title.toLowerCase().contains("chair"); // Check if their affiliation contains the word "Chair" or not.
+                if (chair == true) {
+                    name = individual.select("tbody > tr > td > strong").text();
+                    role = individual.child(0).text();
+                    email = individual.select("tbody > tr > td > a").text();
+                    Chair info = new Chair(name, role, email);
+                    chairs.add(info);
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
```

Comment:
Maybe have some special return in the case of an error? An empty list or null?

Reply:
Makes sense!